packages/integration-test/src/stability.spec.ts

```diff
@@ -48,7 +53,8 @@ describe("stability check, assess a solution, reassess the solution, check all s
     const projects: string[] = projectstring.toString().split(",");
     const solutionPage = `=${solutionPath.split("\\").pop()}`;
     console.log(`checking projects for ${solutionNameTagId}`);
-    for (const project of projects) {
+    for (let i = 0; i < 2 && i < projects.length; i++) {
```

Comment:
We only want to iterate over 2 projects maximum here?

Reply:
yes, running through all projects in nopcommerce may take 3-4 hours,  we currently want to limit the timeout in 2 hours (build exe + run tests).